downloads/swan-lake-release-notes/swan-lake-2201.11.0/RELEASE_NOTE.md

```diff
@@ -344,7 +344,7 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 11 (2201.11.0)
   ```
   $ bal openapi flatten <openapi.yaml>
   ```
-- Introduced the `align` sub-command, which aligns the OpenAPI contract file according to Ballerina's best naming practices. The Ballerina name extensions are added to the schemas which can not be modified directly. The output is a modified OpenAPI contract.
+- Introduced the `align` sub-command, which aligns the OpenAPI contract file according to Ballerina's naming conventions. The Ballerina name extensions are added to the schemas which can not be modified directly. The output is a modified OpenAPI contract.
```

Comment:
```suggestion
- Introduced the `align` sub-command, which aligns the OpenAPI contract file with Ballerina's naming conventions. The Ballerina name extensions are added to the schemas which can not be modified directly. The output is a modified OpenAPI contract.
```

What is "The Ballerina name extensions" btw? Should it be "The Ballerina name extension" or "Ballerina name extensions" instead?

Reply:
They are the ballerina specific extensions(a field in the schema). They are used to change the name or use a annotation to use different names which aligns with Ballerina naming conventions. 

It should be `Ballerina name extensions`. Will change it accordingly